make/protogen.mk

```diff
@@ -1,4 +1,6 @@
 BASE_PATH ?= $(CURDIR)
+# Set to empty string to echo some command lines which are hidden by default.
+SILENT ?= @
```

Comment:
I saw this in the stackrox repo, but I'm wondering why we have this

Reply:
The idea is that you can easily debug a makefile (i.e., print what commands are being executed) by running it with `make SILENT= target`, not having to manually remove the `@` on all the lines that you are interested in.

Reply:
The idea is that you can easily debug a makefile (i.e., print what commands are being executed) by running it with `make SILENT= target`, not having to manually remove the `@` on all the lines that you are interested in.